lib/on_form/saving.rb

```diff
@@ -14,26 +14,21 @@ def transaction(&block)
       end
     end
 
-    def invalid?
-      !valid?
-    end
 
     def save!
       reset_errors
       transaction do
         reset_errors
-        unless run_validations!(backing_model_validations: false)
-          backing_model_instances.each(&:valid?)
-          collect_errors
-          raise ActiveModel::ValidationError, self
-        end
+
+        self.backing_model_validations = false
+        has_form_errors = !run_validations!
+        self.backing_model_validations = true
```

Comment:
Not thrilled about this at all.  At a minimum, it would need to be in a try/ensure block.

Could you explain a bit more about why it is needed?

Reply:
This is setting form object level. If someone updates attributes of form object and chack `valid?` it will exclude backing model validation because we set false while saving. To make sure we are resetting here.